docs/modules/hdfs/pages/usage-guide/operations/pod-disruptions.adoc

```diff
@@ -0,0 +1,105 @@
+
+= Allowed Pod disruptions
+
+You can configure the allowed Pod disruptions for HDFS nodes as described in xref:concepts:operations/pod_disruptions.adoc[].
+
+Unless you configure something else or disable our PodDisruptionBudgets (PDBs), we write the following PDBs:
+
+== JournalNodes
+We only allow a single JournalNode to be offline at any given time, regardless of the number of replicas or `roleGroups`.
+
+== NameNodes
+We only allow a single NameNode to be offline at any given time, regardless of the number of replicas or `roleGroups`.
+
+== DataNodes
+For DataNodes the question how many instances can be unavailable at the same time is a bit harder:
+HDFS stores your blocks on the datanodes.
```

Comment:
Sometimes its DataNodes or datanodes. No opinion but keep it consistent :)

Reply:
I had it lowercase everywhere, Lars changed some entries to PascalCase.
Changed all to PascalCase